### ISSUE TITLE
ADAS_1000 driver

### ISSUE THREAD

**Issue by sokolbiba** [yuanzhoucq/projet-b-pulse#1]
Hi,
Have you tried the adas 1000 driver? Does it work for you with arduino?

**Comment by yuanzhoucq** [comment]
Yes, it works. Before using it you should implement the functions defined in `communication.cpp` according to your micro-controller, just like here I did for Arduino.

**Comment by cornrn** [comment]
在STM32上用硬件SPI HAL库实验过吗？（看名字应该是华人，我就说中文啦！）
我现在在STM32上用的IO口模拟SPI读可以。但是HAL SPI DMA 还是读不起来。时序看起来也没错！！
o(╥﹏╥)o

**Comment by yuanzhoucq** [comment]
抱歉我没有用过你提到的板子和库，不了解相关信息😂

**Comment by cornrn** [comment]
好的，谢谢
